hilo.rb

```diff
@@ -1,24 +1,35 @@
 #!/usr/bin/env ruby
 puts "Welcome to the Hi-Lo game."
```

Comment:
Nice work for providing instruction. You could optionally add a newline character `\n` so you won't need multiple puts. Something like:
`puts "Welcome to the Hi-Lo game.\nGuess a random number from 1 to 100 (inclusive)."`

Reply:
Thanks, done in https://github.com/amykingOW/Hi-Lo-Game/pull/3/commits/141e345a428033321dbf9cf71668f5a16323f7ce.